### ISSUE TITLE
[FEATURE] Support for Hide Save DCs

### ISSUE THREAD

**Issue by ralph2190** [MangoFVTT/fvtt-ready-set-roll-5e#101]
One feature from Better Rolls that I miss is the ability for the GM to hide monster's save DC from the players. A quick peek at the code shows me that at least part of the feature has been implemented but commented out. Will this feature be implemented at some point in the future or has it been cut?

Thank you!

**Comment by ralph2190** [comment]
I should clarify that the option to hide save DC is available per attack/item etc. But it's not available as a global setting.

**Comment by MangoFVTT** [comment]
The Hide Save DC was indeed partially implemented, but put on the backburner to deal with more pressing features/bugfixes. At the time of implementation, it was not correctly functional and thus requires more work to be put in fully.

tl;dr It is one of the pending features I'm still working on. I will update this ticket when progress on the feature is made.🙂

**Comment by GameCrazed** [comment]
Hi! Just wanted to throw my voice in for the ability to hide save DCs. Would be even better if it were possible to show the save type required whilst obscuring the DC itself (ie "Dexterity Save vs DC ???" instead of "Dexterity Save vs DC 10"). I wasn't sure how to voice my interest in this addition without commenting.

Keep up the good work Mango, loving the module so far :)

**Comment by MangoFVTT** [comment]
This feature has been implemented and will be in the next release.

**Comment by ralph2190** [comment]
Maybe I'm reading incorrectly, but I'm still seeing the hide save DC code is still commented out for 1.3.14. Is the fix targeted for a later release?

**Comment by MangoFVTT** [comment]
@ralph2190 This feature has been released with 1.4.0
